CIScripts/Test/Utils/IisTcpTestDockerImageBuild.Tests.ps1

```diff
@@ -0,0 +1,19 @@
+Param (
```

Comment:
If DockerImages directory doesn't belong to tests, then why do these (IisTcpTestDockerImageBuild.Tests.ps1 and IisTcpTestDockerImageBuild.ps1) files are placed here (in Test/Utils)? We suggest moving powershell scripts to directory with dockerfile.

Reply:
What if we want to build this image for another purpose, for example on Linux? I think that Dockerfiles should be seperated from platform-specific scripts which use them. We use PS scripts only for tests but Dockerfiles could be used in other ways.

Reply:
> What if we want to build this image for another purpose, for example on Linux?

I'd say that in that case, scripts for Linux can live alongside ps1 files. And the Dockerfiles are still separated – they live in their specific directories. Similar pattern exists in the `vtest` directory – this directory contains both `ps1` and `sh` scripts for testing, and actual tests are in `vtest/tests` directory. Perhaps this layout would make more sense?
```
CIScripts/
└── Docker
    ├── Dockerfiles
    │   └── iis-tcptest.Dockerfile
    └── Initialize-IisTcpTest.ps1
```

For me, the most important argument for moving the `ps1` file is to move related things together, so the reader of `Initialize-IIsTcpImage` doesn't have to move to totally different place of the repository to locate the Dockerfiles.

Anyway, I don't feels strongly about this directory structure, and if you think it's a good design, it could stay as it is. @jacek-izykowski thoughts?

Reply:
Katarzyna, you may follow your path.